examples/discovery-search-app/cypress/support/utils/defaultHomepage.ts

```diff
@@ -0,0 +1,16 @@
+//TODO: break down the logic in this funtion so that these defaults can be overwritten. We'll probably want to
+// create more util files to make this functionality more intuitive and modular.
+export function mockDefaultHomePage() {
```

Comment:
i'm guessing we went with the utils route over a custom function route so that we might structure this into classes in the future? i'm fine with that approach as it makes it clear what functions are cypress and which ones are our abstractions

Reply:
Yeah, that was my goal when setting this up. I don't think we need as much scaffolding as Tooling has, necessarily, but we could definitely cut down on repeated code by setting up default mocks for certain components.